vmm/vmm.cat

```diff
@@ -2,7 +2,7 @@
  * Copyright (C) Huawei Technologies Co., Ltd. 2024. All rights reserved.
  * SPDX-License-Identifier: MIT
  *
- * Version 0.9.1
+ * Version 0.9.2
```

Comment:
Should the VMM changes be documented in the changelog, or is this an implementation detail?